### DISCUSSION TITLE
Serverless funciton didn't work when deploy

### DISCUSSION THREAD

**Discussion by PrinceM13** [vercel/community#2128] [Help]
### Summary

i have 2 apis, one work on both local and deploy while another one work just local
please help T T

(https://github.com/PrinceM13/edeaf-name-card)

### Additional information

1. this on work just local (on vercel log --> type = request)
/app/api/google-sheet

import { getInfos } from "../../../utils/google-sheet";

export async function GET() {
  const infos = await getInfos();
  return new Response(JSON.stringify(infos));
}

2. this one work on both local and deploy (on vercel log --> type = serverless function)
/app/api/google-sheet/name-cards/[infoId]

import { getInfo } from "../../../../../utils/google-sheet";

export async function GET(request: Request, context: { params: any }) {
  // get params
  const { params: { infoId } } = context;
  const info = await getInfo(infoId);
  return new Response(JSON.stringify(info));
}

**Comment by amyegan** [comment]
Hi @PrinceM13! It looks like you made a code change since asking for help. Did you find a solution?

**Reply by PrinceM13** [reply]
Hi @amyegan can't fix it yet, i have another project that quite similar to this one but work properly both local and deploy so i just try to follow structure of that one but result of this project still same

i just try moving no.1 from /app/api/google-sheet to /app/api/google-sheet/card-names but result still same
(it isn't work so in latest change i already changed it back to previous one)

another project that i mentioned: https://github.com/PrinceM13/note-taking-nextjs
